src/index.ts

```diff
@@ -59,7 +59,7 @@ export default async function GleeAppInitializer() {
   await registerAuth(GLEE_AUTH_DIR)
 
   const parsedAsyncAPI = await getParsedAsyncAPI()
-  const channelNames = parsedAsyncAPI.channelNames()
+  const channelNames = parsedAsyncAPI.channels().all().map(e => e.address())
```

Comment:
You call it `channelNames` but you're getting the addresses. Can't you just get the channels names (ids) and then use it to get the address whenever you need the address?